java/src/test/java/io/cucumber/gherkin/GherkinParserTest.java

```diff
@@ -42,10 +45,43 @@ void use_this_in_readme() {
     @Test
     void test_for_profiler_parser() throws IOException {
         for (int i = 0; i < 1000; i++) {
-            GherkinParser.builder().build().parse(Paths.get("../testdata/good/very_long.feature"));
+            GherkinParser.builder()
+                    .idGenerator(new IdGenerator() {
+                        int id = 0;
+                        @Override
+                        public String newId() {
+                            return Integer.toString(id++);
+                        }
+                    })
+                    .build().parse(Paths.get("../testdata/good/very_long.feature"));
         }
     }
 
+    @Test
+    @Disabled
+    void test_for_profiler_parser_with_summary_statistics() throws IOException {
```

Comment:
Both of these could be replaced with https://github.com/cucumber/gherkin/pull/437.